auth.py

```diff
@@ -26,3 +28,14 @@ def authenticate(username, password):
             return True
 
         return False
+
+
+def require_login():
+     if not current_user:
+          return True
+     return not current_user.is_authenticated and AUTH_ENABLED
```

Comment:
@yukinko-iwasaki this doesn't seem right – shouldn't `require_login()` return False when AUTH_ENABLED is not true, regardless of `current_user`?